.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,9 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
@dupt97 toàn bộ folder .idea là folder của editor em đang dùng, em cần ignore trong dự án của em, ko nên đưa lên git